### ISSUE TITLE
PyInstaller is not working

### ISSUE THREAD

**Issue by satyakidasgupta** [pyinstaller/pyinstaller#3759]
not able to convert following python code into window executable. I am using python 3.6 and openCV 3.4.3. getting following error
for real_module_name, six_module_name in real_to_six_module_name.items():
AttributeError: 'str' object has no attribute 'items'

I have installed PyInstaller with the following command
C:\python36\scripts\pip install https://github.com/pyinstaller/pyinstaller/archive/develop.zip
C:\python36\scripts\pip install --upgrade setuptools

```
import cv2 
import os
import inspect
import os.path

filename = inspect.getframeinfo(inspect.currentframe()).filename
path     = os.path.dirname(os.path.abspath(filename))

os.chdir(path)

id = open('Emp.txt', 'r').read()

writePath = "./Images/"
faceCascade = cv2.CascadeClassifier("haarcascade_frontalface_default.xml")
cam = cv2.VideoCapture(0)

sampleNum = 0

while(True):
    # Capture frame by frame
    ret, frame = cam.read()
    grayFrame = cv2.cvtColor(frame, cv2.COLOR_BGR2GRAY)
    faces = faceCascade.detectMultiScale(grayFrame,scaleFactor=1.5, minNeighbors=5)
    for (x,y,w,h) in faces:
        sampleNum = sampleNum + 1
        cv2.imwrite(writePath + str(id) + "." +str(sampleNum) + ".jpg", grayFrame[y:y+h,x:x+w])

        color = (255,0,0)
        thickness = 2
        cv2.rectangle(frame,(x,y),(x+w, y+h),color, thickness)
    cv2.imshow ("frame",frame)
    cv2.waitKey(2)
    if (sampleNum > 5):
        break

cam.release()
cv2.destroyAllWindows()     

```

**Comment by htgoebel** [comment]
Please try a different version of setuptools. Some versions are not woking properly with PyInstaller, bu I have no time to work out which versions.

**Comment by htgoebel** [comment]
Duplicate of #3806. Should be fixed now in 0ffa574bf87e2e37b0df2a4e2d60cb616f9ff2f2
